pkgs/applications/misc/marktext/default.nix

```
@@ -28,8 +33,16 @@ appimageTools.wrapType2 rec {
     p.xorg.libxkbfile
   ];
 
-  # Strip version from binary name.
-  extraInstallCommands = "mv $out/bin/${name} $out/bin/${pname}";
+  extraInstallCommands = ''
+    # Strip version from binary name.
```

```suggestion
    # rename binary to not contain version
```
striping a binary usually means something else.

---

Definitely, but the comment doesn't talk about stripping the binary though. Looks like a regular string operation, e.g.

 ```python
>>> "binary-version".strip("-version")
'binary'
```

I'm fine changing this if you insist.

---

yeah, I know what you mean but striping something from a binary in a build process usually means removing debugging symbols. I just thought the comment is not that optimal.